src/js/screens/CartContents.js

```diff
@@ -12,18 +12,22 @@ import ArrowButton from '../components/ArrowButton';
 import ProceedButton from '../components/ProceedButton';
 import SectionHeader from '../components/SectionHeader';
 import SecondaryHeader from '../components/SecondaryHeader';
-import { SCREENS } from '../Router';
+import { handleQuickActionsNavigation, SCREENS } from '../Router';
 
 export default class CartContents extends Component {
 	constructor(props) {
 		super(props);
 
 		// If provided through deeplink, add the items to the cart
-    ShoppingCart.addDeeplinkItems(this.props.navigation.getParam('ids', ''));
+		ShoppingCart.addDeeplinkItems(this.props.navigation.getParam('ids', ''));
+	}
+
+	componentDidMount() {
```

Comment:
Why do you need to put this on every screen? You could put on a top component as `App.js`, no?

Reply:
We don't have access in the `App.js` to the navigation props, that's why I did it this way